tests/trading/test_makeOrder.py

```diff
@@ -32,7 +29,7 @@ def test_publicMakeOrder_bid(contractsFixture):
     cash.depositEther(value = 10**17)
     cash.approve(makeOrder.address, 10**17)
 
-    orderID = makeOrder.publicMakeOrder(BID, 10**18, 10**17, market.address, 1, longTo32Bytes(0), longTo32Bytes(0), 7)
+    orderID = makeOrder.publicMakeOrder(contractsFixture.constants.BID(), 10**18, 10**17, market.address, 1, longTo32Bytes(0), longTo32Bytes(0), 7)
```

Comment:
Hmm, I'm not sure about this pattern.  In particular, we reference BID/ASK a _lot_ in tests and having it be this many characters in each case is going to make the tests harder to read.  I don't mind centralizing the constant (rather than putting it at the top of every file) but I would much prefer to do something like `from constants import BID, ASK, YES, NO, LONG, SHORT` so that we can continue to use the pithy names rather than having to fetch.

I recognize that this means we won't be pulling the values out of the contracts, but these values should be stable and the tests _should_ fail if someone changes BID from 0 to 1 (or whatever) so I think there is marginal value in the tests having the data hard coded rather than fetched.

Reply:
I'll see if I can get this to fit within a pattern like that as I agree it would be nice to have shorter versions